### DISCUSSION TITLE
About the files in .devcontainer. How to build/create a docker container for development with these Dockerfiles?

### DISCUSSION THREAD

**Discussion by Corafan** [gdsfactory/gdsfactory#2934] [Q&A] [answered]
In terminal, I open the folder 'gdsfactory/.devcontainer ', and then enter this command 'docker build -t corafan/gdsfactory .'. Althought with the Dockerfile in the folder, I can't build any docker container. In addition, the command can only read the file whose name is Dockerfile, can't read Dockerfile.dev, Dockerfile.dev_minimal and any other files in .devcontainer. So in this situation, I don't know how to  build a docker container for development. Maybe my approach is wrong, but I really don't have any idea to deal with this problem, I mean ' how to build a docker container for development'. Please help me, thank you all every much!

**Comment by joamatab** [comment, marked as answer]
the container is already built and available in github

https://github.com/gdsfactory/gdsfactory/pkgs/container/gdsfactory

**Reply by Corafan** [reply]
Thank you for you kind reply! Best wishes!
